projects/manager/src/app/datasets/loading-info/gdal-dataset-parameters/gdal-dataset-parameters.component.html

```diff
@@ -0,0 +1,138 @@
+<form [formGroup]="form">
+    <mat-form-field appearance="fill">
+        <mat-label>File Path (relative to upload or volume)</mat-label>
+        <input matInput type="text" formControlName="filePath" />
+    </mat-form-field>
+    <mat-form-field appearance="fill">
+        <mat-label>Raster band channel</mat-label>
+        <input matInput type="number" formControlName="rasterbandChannel" />
+    </mat-form-field>
+
+    <div class="flex-container" formGroupName="geoTransform">
+        <ng-container formGroupName="originCoordinate">
+            <mat-form-field appearance="fill">
+                <mat-label>Origin X</mat-label>
+                <input matInput type="number" formControlName="x" />
+            </mat-form-field>
+            <mat-form-field appearance="fill">
+                <mat-label>Origin Y</mat-label>
+                <input matInput type="number" formControlName="y" />
+            </mat-form-field>
+        </ng-container>
+        <mat-form-field appearance="fill">
+            <mat-label>Pixel Size X</mat-label>
+            <input matInput type="number" formControlName="xPixelSize" />
+        </mat-form-field>
+        <mat-form-field appearance="fill">
+            <mat-label>Pixel Size Y</mat-label>
+            <input matInput type="number" formControlName="yPixelSize" />
+        </mat-form-field>
+    </div>
+
+    <div class="flex-container">
+        <mat-form-field appearance="fill">
+            <mat-label>Width (pixels)</mat-label>
+            <input matInput type="number" formControlName="width" />
+        </mat-form-field>
+        <mat-form-field appearance="fill">
+            <mat-label>Height (pixels)</mat-label>
+            <input matInput type="number" formControlName="height" />
+        </mat-form-field>
+    </div>
+
+    <div class="flex-container">
+        <mat-form-field appearance="fill">
+            <mat-label>File not found handling</mat-label>
+            <mat-select formControlName="fileNotFoundHandling">
+                <mat-option *ngFor="let handling of FileNotFoundHandling" [value]="handling">
+                    {{ handling }}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <mat-form-field appearance="fill">
+            <mat-label>No data value (leave empty if there is none)</mat-label>
+            <input matInput type="number" formControlName="height" />
+        </mat-form-field>
+    </div>
+
+    <div formArrayName="propertiesMapping" *ngFor="let control of form.controls.propertiesMapping.controls; let i = index">
```

Comment:
When adding a second time step there are placeholders for properties and so on. The are confusing: 
![image](https://github.com/geo-engine/geoengine-ui/assets/1027102/2a251da2-1857-4832-9f24-f49f0bdb9f25)

Reply:
c1906386f491dda8fae772f67c2cde2ae2420fea